spec/classes/vault_spec.rb

```diff
@@ -197,9 +197,17 @@
           it { is_expected.not_to contain_file_capability('vault_binary_capability') }
 
           context 'when managing file capabilities' do
-            let(:params) { { manage_file_capabilities: true } }
+            let(:params) do
```

Comment:
Instead of duplicating the params from the parent context can you merge them like:

```
let(:params) do
  super().merge(manage_file_capabilities: true)
end
```